movie-browser/src/App.js

```diff
@@ -1,7 +1,36 @@
-import React from 'react';
+import React, { Component } from 'react';
+import MovieList from './components/MovieList';
+import axios from 'axios';
 
-const App = () => {
-  return <div>Movie Browser</div>;
-};
+const URL = 'https://ghibliapi.herokuapp.com/films';
+
+class App extends Component {
+  state = {
+    movies: [],
+  };
+
+  componentDidMount() {
+    axios
```

Comment:
what if some other component needs to make this api call?
what if u need to do another api call in here but with different parameters?

This is not reusable atm. You should abstract the definition of the call to a function on some other file. 

E.g file api.js, and a method `fetchMovies` that does this call.

Reply:
noted, can be imported and used to make multiple calls in this or any other component, for any endpoint.